rest/route/admin_banner_test.go

```diff
@@ -0,0 +1,64 @@
+package route
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen/rest/data"
+	"github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"golang.org/x/net/context"
+)
+
+type AdminBannerRouteSuite struct {
```

Comment:
seems like this could all be a single case, I'm not sure that the suite is really getting you anything here?

Reply:
I'm using it to hold some state info, but that can also be done without a suite. However I feel like having it in a suite is a pretty good way to hold that data. Let me know if you disagree and I can change it

Reply:
I think that unless there are multiple cases or multiple different setups for the same case, this just makes it more confusing to understand what the test is doing, so I would change it, unless you have the expectation that this will grow in the future.

Reply:
Done